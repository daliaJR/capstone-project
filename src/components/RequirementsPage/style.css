@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700&display=swap');

body {

  font-family: 'Poppins';
  background-color: #fcfcfc;

}


.main{
    display: flex;
}

.content .headline{
    font-size: 50px;
}

.content{
    text-transform: capitalize;
    margin-left: 150px;
    padding: 10px;
}

.heading{
 
    color: black;
    font-size: 20px;
    font-weight: 500;
   margin-top: 10px;
   margin-bottom: 5px;
}




.sections{
    font-weight: 500;
    color: rgba(0, 0, 0, 0.54);
    margin-top: 20px;
    margin-bottom: 20px;
}

.requirments{
    
 
    font-size: 35px;
    font-weight: 500;
    margin-top:40px ;
}

ul{
    font-weight: 500;
    color: rgba(0, 0, 0, 0.54);
    text-transform: capitalize;
    list-style:disc;
    list-style: disc;
}
li{
    list-style:disc;
 padding: 5px;
}

.frame{
    margin-right: 100px;
    margin-left: 80px;
    margin-top: 60px;
}

.btn{
    margin-top: 30px;
    margin-bottom: 40px;
    display: flex;
    background-color: aqua;
    width: 202px;
    height:60px;
   align-content: center;
 justify-content: center;
    align-content: center;
    font-family: 'Poppins';
    font-style: normal;
    font-size: 24px;
   
}

button{
    text-align: center;
}