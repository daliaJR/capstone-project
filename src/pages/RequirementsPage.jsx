import React from 'react';
import { data } from '../staticData/requirementsPage';
import { ReactComponent as Frame } from '../images/Frame.svg';
import '../styles/RequirementPage.css';

export default function RequirementsPage() {
  return (
    <div className="max-w-7xl mx-auto main font-poppins py-16">
      <div className="flex flex-col lg:flex-row space-x-10">
        <div className="content">
          <h1 className="headline uppercase">Why work with Healing?</h1>

          <div className="sections">
            <h2 className="heading uppercase">Reliable Income</h2>
            <p className="text-lg">{data.reliableIncome}</p>
          </div>

          <div className="sections">
            <h2 className="heading uppercase">Focus on Counseling</h2>
            <p className="text-lg">{data.focusonCounseling}</p>
          </div>

          <div className="sections">
            <h2 className="heading uppercase">Focus on Counseling</h2>
            <p className="text-lg">{data.focusonCounseling}</p>
          </div>

          <div className="req mb-8">
            <h1 className="requirments uppercase">Requirements</h1>
            <ul className="pl-5">
              <li>Licensed by a State Board to provide counseling</li>
              <li>
                Experience in counseling for adults, couples, and/or teens
              </li>
              <li>Excellent writing skills</li>
              <li>Reliable Internet connection</li>
              <li>Currently residing in the US</li>
            </ul>
          </div>
          <div>
            <button type="button" className="blue_button">
              Get started
            </button>
          </div>
        </div>

        <div className="frame self-center lg:self-start">
          <Frame />
        </div>
      </div>
    </div>
  );
}
