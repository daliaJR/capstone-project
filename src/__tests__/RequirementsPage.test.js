import * as React from 'react';
import * as renderer from 'react-test-renderer';
import RequirementsPage from '../pages/RequirementsPage'

it('Requirement Page renders correctly', () => {
  const tree = renderer.create(<RequirementsPage />).toJSON();
  expect(tree).toMatchSnapshot();
});
