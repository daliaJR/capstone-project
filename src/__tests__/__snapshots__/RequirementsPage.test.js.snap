// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Requirement Page renders correctly 1`] = `
<div>
  <div
    className="main"
  >
    <div
      className="content"
    >
      <h1
        className="headline"
      >
        Why work with Healing?
      </h1>
      <div
        className="sections"
      >
        <h2
          className="heading"
        >
          Reliable Income
        </h2>
        <p
          className="paragraph"
        >
          Over 10,000 people sign up on BetterHelp every day looking for a counselor to help with lifes challenges. BetterHelp can be your main source of income or a supplement to your current work.
        </p>
      </div>
      <div
        className="sections"
      >
        <h2
          className="heading"
        >
          Focus on Counseling
        </h2>
        <p
          className="paragraph"
        >
          No need to worry about costs from acquiring clients, billing, support or operations. Let us handle the fees and paperwork so you can focus on what you do best!
        </p>
      </div>
      <div
        className="sections"
      >
        <h2
          className="heading"
        >
          Focus on Counseling
        </h2>
        <p
          className="paragraph"
        >
          No need to worry about costs from acquiring clients, billing, support or operations. Let us handle the fees and paperwork so you can focus on what you do best!
        </p>
      </div>
      <div
        className="req"
      >
        <h1
          className="requirments"
        >
          Requirements
        </h1>
        <ul>
          <li>
            Licensed by a State Board to provide counseling
          </li>
          <li>
            Experience in counseling for adults, couples, and/or teens
          </li>
          <li>
            Excellent writing skills
          </li>
          <li>
            Reliable Internet connection
          </li>
          <li>
            Currently residing in the US
          </li>
        </ul>
      </div>
      <div
        className="btn"
      >
        <button
          type="button"
        >
          Get started
        </button>
      </div>
    </div>
    <div
      className="frame"
    >
      <svg>
        Frame.svg
      </svg>
    </div>
  </div>
</div>
`;
